generic_petg.xml.fdm_material

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Generic PLA profile. Serves as an example file, data in this file is not correct.
+-->
+<fdmmaterial xmlns="http://www.ultimaker.com/material">
+    <metadata>
+        <name>
+            <brand>Generic</brand>
+            <material>PETG</material>
+            <color>Generic</color>
+        </name>
+        <GUID></GUID>
```

Comment:
I believe it is just a bit of Python code:

```
import uuid

uuid.uuid4()
```

or something along those lines.